internal/readcloser/limit.go

```diff
@@ -8,18 +8,18 @@ import "io"
 // LimitReadCloser is a LimitReader extension that returns a ReadCloser
 // that reads from r but stops with EOF after n bytes.
 func LimitReadCloser(r io.ReadCloser, n int64) io.ReadCloser {
-	return &limitedReadCloser{io.LimitReader(r, n), r}
+	return &LimitedReadCloser{io.LimitReader(r, n), r}
```

Comment:
Let's leave this change for later.

Reply:
what about the other misc changes in this commit: https://github.com/storj/storj/pull/2196/commits/24678d42e919c6bd00e33f35ff614563c961abc4. Can I just revert the whole thing?

Reply:
Yeah, go ahead. One of them is already fixed and merged in master, and I'll get the other.